### PR TITLE
#160451586 Add and configure travis.yml for CI

### PR DESCRIPTION
**Pull request by andela-phemmy**
#### What does this PR do?
  Add and configure travis.yml for continous integration
#### Description of Task to be completed?
  1. Add travis.yml file
  2. configured travis to include node in build system
  3. updated readme to include travis badge
#### Any background context you want to provide?
   Continuous integration is needed in other to check the integrity of a system remotely.
#### What are the relevant pivotal tracker stories?
   [Pivotal tracker story](https://www.pivotaltracker.com/story/show/160451586 "Pivotal Tracker Board")

### CODE REVIEW

**Review by koiic — APPROVED** [review]
Looks good to me

**Review by Victor-Ugwueze — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/andela/lotus-ah.svg?branch=develop)](https://travis-ci.org/andela/lotus-ah)
 Authors Haven - A Social platform for the creative at heart.
```
Comment (KaiserPhemi) [inline README.md]: #### Nitpick
This should be on a new line:
screen shot 2018-09-25 at 2 18 32 pm

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+language: node_js
+
+node_js:
+  - 'stable'
+
+services:
+  - postgresql
+
+before_script:
+  - psql -c 'create database travis_ci_test;' -U postgres
```
Comment (KaiserPhemi) [inline .travis.yml]: #### Nitpick
New line after the last line. For context check [here](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]
#### Nitpick
Implement feedback.

File: README.md (hunk 1 above)
Comment (andela-phemmy) [reply README.md]: @KaiserPhemi, I have implemented the feedback and pushed

**Review by KaiserPhemi — APPROVED** [review]

**Comment by KaiserPhemi** [comment]
#### Critical
@andela-phemmy 
Please fix the merge conflicts

**Comment by andela-phemmy** [comment]
@KaiserPhemi, I have fixed the merge conflicts

**Pull request merged**